gax-grpc/src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -170,6 +175,11 @@ public Builder applyToAllUnaryMethods(
       return getStubSettingsBuilder().deleteOperationSettings();
     }
 
+    /** Returns the builder for the settings used for calls to deleteOperation. */
+    public UnaryCallSettings.Builder<WaitOperationRequest, Operation> waitOperationSettings() {
```

Comment:
test

Reply:
Ditto.

Reply:
I still think there should be a test of this methjod.